src/clarity-angular/popover/signpost/signpost.module.ts

```diff
@@ -9,12 +9,14 @@ import {CommonModule} from "@angular/common";
 import {ClrCommonPopoverModule} from "../common/popover.module";
 import {ClrIconModule} from "../../icon/icon.module";
 import {SIGNPOST_DIRECTIVES} from "./index";
+import {ClrConditonalModule} from "../../utils/conditional/conditional.module";
 
 @NgModule({
     imports: [
         CommonModule,
         ClrCommonPopoverModule,
-        ClrIconModule
+        ClrIconModule,
+        ClrConditonalModule
```

Comment:
Same thing here, except in this case we do want the `ClrIconModule` imported. But the `ClrConditonalModule` should be exported, not imported.